Cargo.toml

```diff
@@ -249,6 +249,11 @@ ndk-sys = "0.6"
 [patch.crates-io]
 wgpu = { path = "./wgpu" }
 
+# https://github.com/Traverse-Research/gpu-allocator/pull/258
+gpu-allocator = { git = "https://github.com/Traverse-Research/gpu-allocator", rev = "955b13a" }
+# https://github.com/Xudong-Huang/generator-rs/pull/72
+generator = { git = "https://github.com/MarijnS95/generator-rs", rev = "e4afb28" }
```

Comment:
Looks like @Xudong-Huang [kept ignoring](https://github.com/Xudong-Huang/generator-rs/pull/72#issuecomment-2645924543) my PR and silently jumped straight to `0.60` in https://github.com/Xudong-Huang/generator-rs/commit/250f96ae7ba5e916ed2a83d3c216568f5f810e24.